packages/app/package.json

```diff
@@ -79,5 +79,10 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "nx": {
+    "implicitDependencies": [
+      "@packages/data-context"
```

Comment:
This and the one for frontend-shared should not need to be implicit... we import stuff from data-context in these packages so I'm not sure why Nx doesn't think that they depend on eachother, but I couldn't get the build to work unless I added these